### DISCUSSION TITLE
Heartbeat

### DISCUSSION THREAD

**Discussion by canacar7** [OpenDDS/OpenDDS#3858] [Q&A] [answered]
How I create heartbeat on DevGuideExamples/DCPS/Messenger?

**Comment by mitza-oci** [comment, marked as answer]
Please expand on and clarify your question.

**Reply by canacar7** [reply]
If the subscriber drops during communication, I want to understand whether it is alive or dropped from the line ?

**Reply by mitza-oci** [reply]
In DDS this is the Liveliness QoS policy.
